Global.xcconfig

```diff
@@ -12,6 +10,9 @@ MS_SIM_ARCHS = x86_64 i386
 ARCHS = $(MS_ARM_ARCHS)
```

Comment:
We should define architectures directly in an xconfig file dedicated to all iOS targets and do the same for macOS. This will prevent having mac config variables showing up in iOS projects.